docs/languages/csharp/expression-trees.md

```diff
@@ -0,0 +1,65 @@
+﻿# Expression Trees
+
+If you have used LINQ, you have experience with a rich library
+where `Func` and `Func<T>` are part of the API. (If you are not familiar
+with LINQ, you probably want to read [that tutorial](linq.md) and
+the tutorial on [lambda expressions](lambda-expressions.md) before this one.)
+*Expression Trees* provide richer interaction with the arguments that
+are functions.
+
+Using a typical LINQ query, and function arguments are represented
```

Comment:
I'm not quite sure what this sentence is saying.  Is it saying that a lambda expression is a delegate that the compiler creates?